src/blob.rs

```diff
@@ -225,9 +225,10 @@ impl<'a> BlobObject<'a> {
         context: &'a Context,
         name: String,
     ) -> std::result::Result<BlobObject<'a>, BlobError> {
-        let name: String = match name.starts_with("$BLOBDIR/") {
-            true => name.splitn(2, '/').last().unwrap().to_string(),
-            false => name,
+        let name: String = if name.starts_with("$BLOBDIR/") {
+            name.splitn(2, '/').last().unwrap().to_string()
+        } else {
+            name
```

Comment:
fwiw i somewhat disagree with this clippy suggestion.   i like match expressions better than if's often as they are more explicit.

Reply:
also it's one line less ;)

Reply:
Reverted in 4 places and allowed this as @flub also disagreed at some point I think.